dg/setting-up.md

```diff
@@ -13,16 +13,17 @@ This section guides you through the steps required to set up your computer for d
 ## Getting started
 
 **Prerequisites:**
-* [Node.js 12](https://nodejs.org/en/blog/release/v12.20.0/) -- (run `node -v` in your OS terminal to check the version of Node on your computer)
+* [Node.js 14](https://nodejs.org/en/blog/release/v14.21.2/) -- (run `node -v` in your OS terminal to check the version of Node on your computer)
 
 **Recommended:**
 * IDE: Visual Studio Code
 
 **Steps:**
 1. Fork this repository into your Github account.
-1. Clone the forked repository into your computer.
-1. Install dependencies with npm: Run `npm install`.
-1. Compile and start the application: Run `npm start`.
+2. Clone the forked repository into your computer.
+3. Install dependencies with npm: Run `npm install`
+    - *a list of warnings is expected and no action needs to be taken before proceeding.*
```

Comment:
I am not sure if this line should be added to the docs. Ideally there should not be any warnings, but there are because many of our dependencies are out of date.

Reply:
I believe line25 is important for those beginners who are unsure about whether this should be expected. They may start running `npm audit fix` as instructed by the npm while they may not be required to do so.

I think we can remove this line when our dependencies are updated completely, but at a much later date.

Reply:
@Echomo-Xinyu Perhaps we can specify that warnings for out-of-date dependencies are expected

Reply:
Noted!